packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/lib_router/models/route_model.dart

```diff
@@ -0,0 +1,70 @@
+{{> licence.dart }}
+
+class RoutePath {
+  static const splash = '/splash';
+  static const counter = '/';
+  static const notifications = 'notifications';
+  static const login = 'login';
+  static const enterMessage = 'enterMessage';
+  static const items = 'items';
+  static const itemDetails = ':id';
+}
+
+enum RouteModel {
+  splash(
+    pathName: RoutePath.splash,
+    fullPath: '/splash',
+    permissionName: 'SplashRoute',
```

Comment:
As discussed those strings needs to be moved as const in the permissions lib

Reply:
Done